src/__swaps__/screens/Swap/providers/swap-provider.tsx

```diff
@@ -630,46 +635,23 @@ export const SwapProvider = ({ children }: SwapProviderProps) => {
       return { icon: '􀕹', label: review, disabled: true };
     }
 
-    const inputAsset = internalSelectedInputAsset.value;
-    const sellAmount = (() => {
-      const inputAmount = SwapInputController.inputValues.value.inputAmount;
-      if (!quote.value || 'error' in quote.value) return inputAmount;
-      return quote.value.sellAmount.toString();
```

Comment:
separately, relying on the quote as a source of truth for the sell amount can cause the button to flicker because the quote can still technically have a value / not have an error, even though it is now out of date with the input